packages/core/src/util/config.ts

```diff
@@ -107,6 +107,9 @@ export class Config {
   static getHl7IncomingMessageBucketName(): string {
     return getEnvVarOrFail("HL7_INCOMING_MESSAGE_BUCKET_NAME");
   }
+  static getHl7RawMessageBucketName(): string {
+    return getEnvVarOrFail("HL7_RAW_MESSAGE_BUCKET_NAME");
+  }
```

Comment:
if its operating at the same layer as the lahie + alohr ingestion buckets, lets follow a naming pattern to indicate that.

Maybe `MLLP_SERVER_INGESTION_BUCKET_NAME` or something similar.